tests/job-submission/17-remote-localtime.t

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
+# Copyright (C) 2008-2018 NIWA
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#-------------------------------------------------------------------------------
+# Test remote host job log NN link correctness.
```

Comment:
This seems to be the description from the ``02-job-nn-remote-host.t`` test which I guess was adapted to run with local not UTC time. Ideally update this heading for clarity before we merge.

Reply:
Another copy-and-paste failure. Sorry.

Reply:
Comment fixed.